net-misc/tigervnc/tigervnc-1.12.0-r1.ebuild

```diff
@@ -185,3 +193,14 @@ src_install() {
 		rm -r "${ED}"/usr/share/man/man8 || die
 	fi
 }
+
+pkg_postinst() {
+	if use server; then
+		elog "In order for Xsession to work properly, you also need"
+		elog "to install a display manager.  Please emerge one of:"
+		elog "  gnome-base/gdm"
+		elog "  x11-misc/lightdm"
+		elog "  x11-misc/sddm"
+		elog "  x11-misc/slim"
+	fi
```

Comment:
This is what I said above, we need a valid Xsession file.  Any of these packages provide one, but I don't think we should depend on them.  One can actually make it work with the one provided by xinit, only it's harder and more arcane :)